### ISSUE TITLE
Abductor All Access Organ(Anagraphic electro-scrambler) Does not give AA after Activation

### ISSUE THREAD

**Issue by Taylorkills2** [tgstation/tgstation#55237]
## Reproduction: 
Be abductor, give someone the gland, mind control them to recall shuttle for you. Watch as they have normal access and can't escape the place you released them in.
The gland is supposed to give them innate AA once you mind control them with the MID, but both times I've done it they just failed to open command doors/sliding glass panes.

**Comment by ATH1909** [comment]
er, no, that's not how it works. glands don't activate upon mind controlling someone. they activate after a delay after you're either beamed down to the station (if the gland was installed the special abductor way) or after they're inserted into you (if the gland was installed via the organ manipulation surgery). after being activated, the gland will automatically activate again after another delay (and this process will repeat until the gland runs out of charges).

**Comment by ATH1909** [comment]
the delay for anagraphic electro-scramblers is 1-2 minutes, btw.

**Comment by Taylorkills2** [comment]
Hm, a misinterpretation with this gland then. I know the others do their thing randomly. Apologies, the times I've used it they didn't have AA any times I observed them or when I was mind controlling them.
I understand how the other glands worked for the most part, didn't know they had limited activations(and deactivations) though.
I'll give it out to perma prisoners and see if they ever manage to walk out. Thanks.
